graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/fetchers/deployed/UpdateFetcher.java

```diff
@@ -44,15 +47,14 @@ public UpdateFetcher(UpdateModel model, MappingModel mappingModel) {
   }
 
   @Override
-  protected Map<String, Object> get(
+  protected Object get(
```

Comment:
Isn't it better to have a more strict contract and always return `Map<String, Object>` instead of two cases: (boolean and map)?

Reply:
But then it wouldn't be possible to implement this signature anymore:
```
updateUser(user: UserInput!): Boolean
```
I think we need to support it, it's the most straightforward and logical for non-conditional queries.